pastel/client.go

```diff
@@ -75,6 +75,25 @@ func (client *client) IDTickets(ctx context.Context, idType IDTicketType) (IDTic
 	return tickets, nil
 }
 
+// TicketOwnership implements pastel.Client.TicketOwnership
+func (client *client) TicketOwnership(ctx context.Context, txID, pastelID, passphrase string) (string, error) {
+	ownership := OwnershipTicket{}
```

Comment:
OwnershipTicket is not used in other code, then should define local here

Reply:
Fixed. thanks.